Libraries/LibWasm/AbstractMachine/BytecodeInterpreter.cpp

```diff
@@ -25,6 +25,15 @@
 
 using namespace AK::SIMD;
 
+// ASAN allocates frames for all the not-explicitly-tail-called functions,
+// which blows out the stack. So disable direct threading when ASAN is enabled.
+// We use the explicit annotation on clang, so allow it there.
+#if defined(HAS_ADDRESS_SANITIZER) && !defined(AK_COMPILER_CLANG)
+constexpr static auto should_try_to_use_direct_threading = false;
+#else
+constexpr static auto should_try_to_use_direct_threading = true;
+#endif
```

Comment:
Didn't you add the missing tail call annotations, or are there still some missing?

Reply:
None missing, but gcc<15 doesn't have the annotation.

Reply:
asking because of the ASAN part still there, not the GCC portion

Reply:
Ah sorry, misread/interpreted the `#if` condition, mb